lib/assessment.rb

```diff
@@ -4,37 +4,38 @@ class Assessment
   # Write ruby code to return the longest entry in the
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
-  def longest(enum)
-    # Your code goes here
+  def longest(enum) { |a, b| a.length <=> b.length }
+    enum.max
   end
 
+
   # Question 2
   # Write ruby code to return the sum of entries in the
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.length
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    numbers.some?.odd?
```

Comment:
Close! `some` is JavaScript - the analogue in Ruby is `any?`. Also, the syntax to pass `odd?` into it is a little off.
`numbers.any?(&:odd?)`